src/server/index.jsx

```diff
@@ -65,6 +66,7 @@ const constructDataFilePath = ({
   switch (pageType) {
     case 'frontpage':
     case 'mostRead':
+    case 'idx':
```

Comment:
Don't think you need this in here as you're constructing the path below!

Reply:
True, removed this:)